pom.xml

```diff
@@ -71,14 +71,14 @@
         <jacoco.version>0.8.13</jacoco.version>
         <kubernetes-client.version>7.3.0</kubernetes-client.version> <!-- Please check with Java Operator SDK team before updating -->
         <rest-assured.version>5.5.2</rest-assured.version>
-        <hibernate-orm.version>6.6.15.Final</hibernate-orm.version> <!-- WARNING when updating, also align the versions below -->
-        <jakarta.persistence-api.version>3.1.0</jakarta.persistence-api.version> <!-- version controlled by Hibernate ORM's needs -->
+        <hibernate-orm.version>7.0.0.Final</hibernate-orm.version> <!-- WARNING when updating, also align the versions below -->
+        <jakarta.persistence-api.version>3.2.0</jakarta.persistence-api.version> <!-- version controlled by Hibernate ORM's needs -->
         <antlr.version>4.13.0</antlr.version> <!-- version controlled by Hibernate ORM's needs -->
         <bytebuddy.version>1.15.11</bytebuddy.version> <!-- version controlled by Hibernate ORM's needs -->
-        <hibernate-commons-annotations.version>7.0.3.Final</hibernate-commons-annotations.version> <!-- version controlled by Hibernate ORM's needs -->
-        <hibernate-reactive.version>2.4.8.Final</hibernate-reactive.version> <!-- highly sensitive to Hibernate ORM upgrades -->
+        <hibernate-models.version>1.0.0.CR3</hibernate-models.version> <!-- version controlled by Hibernate ORM's needs -->
```

Comment:
Is the final version of ORM 7 using a CR3 of hibernate-models?

I don't see a Final on Central so I suppose it's the case but it's a bit odd?

Reply:
Yeah it's odd -- I'll bring this up with Steve.